content/docs/command-reference/commit.md

```diff
@@ -36,23 +36,26 @@ further detailed below.
 - It's always possible to manually execute the source code used in a stage
   without DVC (outputs should be unprotected or removed first in certain cases,
   see `dvc unprotect`). Once a desirable result is reached, use `dvc add` or
-  `dvc commit` as appropriate to update DVC-files and store changed data to the
-  cache.
+  `dvc commit` as appropriate to update the corresponding stage (or `.dvc` file)
+  and store changed data to the cache.
```

Comment:
Same. Not quite right.

Reply:
Here also it is was previously mentioned that `commit` updates files but that already happened after running `run` or `repro`. It just saves data to cache. Have pushed new update.

Reply:
It does more than that.